vagga.yaml

```diff
@@ -0,0 +1,28 @@
+containers:
+    gdg_dev:
+      setup:
+      - !Ubuntu trusty
+      - !UbuntuUniverse
+      - !Install [software-properties-common, curl, git, openssl, libssl-dev, libpq-dev, ca-certificates, build-essential, wget, python-pip, python-dev, mercurial]
+      - !Sh |
+          curl -sL https://deb.nodesource.com/setup_5.x | sudo -E bash -
+          sudo add-apt-repository -y ppa:fkrull/deadsnakes
+          sudo apt-get update
+      - !Install [python3.5, nodejs]
+      - !Sh |
+          pip install virtualenv
+commands:
+  run: !Supervise
+    description: Run app - testing status
+    children:
+      env: !Command
+        container: gdg_dev
+        run: |
+          export OAUTHLIB_INSECURE_TRANSPORT=1
```

Comment:
I'd replace that with exporting `.exports` (which is `.gitignore`d and should be created during setup)